playwright/tests/pageTypes.spec.ts

```diff
@@ -0,0 +1,63 @@
+import * as faker from "faker";
+import { PAGE_TYPES } from "@data/e2eTestData";
+import { PageTypesPage } from "@pages/pageTypesPage";
+import { test, expect } from "@playwright/test";
+
+test.use({ storageState: "./playwright/.auth/admin.json" });
+const pageTypeName = `e2e-page-type-${faker.datatype.number()}`;
+
+test("TC: SALEOR_187 As an admin user I can create page type @e2e @page-type", async ({
+    page,
+}) => {
+    const pageTypePage = new PageTypesPage(page);
+
+    await pageTypePage.gotoPageTypeListPage();
+    await pageTypePage.clickCreatePageTypeButton();
+    await pageTypePage.typePageTypeName(pageTypeName);
+    await pageTypePage.clickSaveButton();
+    await pageTypePage.expectSuccessBanner();
```

Comment:
Add assertions please - currently you do not check if the page with correct details has been created